spec/dynamoid/dumping_spec.rb

```diff
@@ -1619,7 +1619,8 @@ def self.dynamoid_field_type
       obj = klass.create(image: binary_value)
 
       expect(reload(obj).image).to eql(binary_value)
-      expect(raw_attributes(obj)[:image]).to eql(Base64.strict_encode64(binary_value))
+      expect(raw_attributes(obj)[:image].class).to eql(StringIO)
+      expect(raw_attributes(obj)[:image].read).to eql(binary_value)
```

Comment:
Probably the test title is not accurate anymore.

Reply:
Another concern is String encoding. When document is loaded with manual `Base64.strict_encode64` call result has US-ASCII encoding. We need to ensure it isn't changed with this change. Probably we need new tests for this.